src/librustc_codegen_llvm/base.rs

```diff
@@ -715,7 +714,7 @@ pub fn codegen_crate<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
                              rx: mpsc::Receiver<Box<Any + Send>>)
                              -> OngoingCodegen {
 
-    check_for_rustc_errors_attr(tcx);
+    ::rustc_codegen_utils::check_for_rustc_errors_attr(tcx);
```

Comment:
What's the advantage to not importing this?

Reply:
Nothing